client/js/components/sidebar/sidebar.js

```diff
@@ -0,0 +1,13 @@
+var React = require('react');
+
+var Sidebar = React.createClass({
```

Comment:
Are we going to make a reusable sidebar or one for each view? Are they logically separate enough to merit a different component?